src/ontology/uberon-edit.obo

```diff
@@ -222010,6 +222010,16 @@ relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:0003687 ! foramen magnum
 property_value: depicted_by http://locatelambda.org/wp-content/uploads/2018/02/opisthion.jpg xsd:string
 
+[Term]
+id: UBERON:7500121
+name: paroccipital process
+def: "An occasional process of bone extending downward from the jugular process of the occipital bone." [https://medical-dictionary.thefreedictionary.com/paroccipital+process]
```

Comment:
Re: the use of "occasional":

"It is present in 0.5%−2% of the population..."
From https://doi.org/10.1016/j.wneu.2018.06.056

Reply:
I may change this to a comment and remove the "rare" and "occasional" altogether, as it is only rare in humans.